### ISSUE TITLE
No Typescript support

### ISSUE THREAD

**Issue by iisdan** [mcohen01/node-quickbooks#162]

**Comment by gdbjohnson** [comment]
5 years later.  I guess this repo is dead.  Looks like there's a PR open from 2019...

**Comment by luskin** [comment]
+1 time for typescript

**Comment by mcohen01** [comment]
> 5 years later. I guess this repo is dead. Looks like there's a PR open from 2019...

Last [PR that was merged](https://github.com/mcohen01/node-quickbooks/pull/225) was a year ago, so it's not quite that bad. Pull requests welcome 😁

**Comment by AurelienGasser** [comment]
Kind of crazy that we rely on the work of unpaid volunteers when Intuit, the owner of QuickBooks, [has a revenue of $16.3 billion](https://investors.intuit.com/news-events/press-releases/detail/1202/intuit-reports-strong-fourth-quarter-and-full-year-results-sets-fiscal-2025-guidance-with-double-digit-revenue-and-earnings-growth)

**Comment by Tyler-Petrov** [comment]
Yeah, it seems like it would be worth it to them...  Any updates here?
